### PR TITLE
Added org.hikounomizu.HikouNoMizu

### PR DESCRIPTION
**Pull request by hnm-duncan**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
_device=all is used to access joysticks_
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.hikounomizu.HikouNoMizu

File: org.hikounomizu.HikouNoMizu.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            sha256: a8a92f95dbb80a3d66b8dc8e54634d583a2e956f41dbd8566679dbb53cae115b
+
+          - type: file
+            path: org.hikounomizu.HikouNoMizu.svg
+
+          - type: file
+            path: org.hikounomizu.HikouNoMizu.desktop
+
+          - type: file
+            path: org.hikounomizu.HikouNoMizu.appdata.xml
```
Comment (LeandroStanger) [inline org.hikounomizu.HikouNoMizu.yaml:47]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by hnm-duncan** [comment]
bot, build org.hikounomizu.HikouNoMizu

File: org.hikounomizu.HikouNoMizu.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            dest-filename: autogen.sh
+            commands:
+                - 'autoreconf -vfi'
+
+    - name: hikounomizu
+      buildsystem: cmake
+      post-install:
+          - 'install -Dm644 org.hikounomizu.HikouNoMizu.svg /app/share/icons/hicolor/scalable/apps/org.hikounomizu.HikouNoMizu.svg'
+          - 'install -Dm644 org.hikounomizu.HikouNoMizu.desktop /app/share/applications/org.hikounomizu.HikouNoMizu.desktop'
+          - 'install -Dm644 org.hikounomizu.HikouNoMizu.metainfo.xml /app/share/metainfo/org.hikounomizu.HikouNoMizu.metainfo.xml'
```
Comment (barthalion) [inline org.hikounomizu.HikouNoMizu.yaml:33]: Ideally these 3 files should come from your source code tarball.

**Pull request closed without merging**